src/pages/_app.tsx

```diff
@@ -104,14 +100,7 @@ export default function App({
 	 * TODO: refactor this so that pageProps.layoutProps is the only place where
 	 * layoutProps come from.
 	 */
-	const allLayoutProps = {
-		theme: Component.theme,
-		...pageProps.layoutProps,
-		// @ts-expect-error - layoutProps is inferred from `fetchLayoutProps`,
-		// which current resolves to `unknown | null`.
-		// Spread types may only be created from object types.
-		...layoutProps,
-	}
+	const allLayoutProps = { theme: Component.theme, ...pageProps.layoutProps }
```

Comment:
I've removed `layoutProps` entirely. The variable name is a bit misleading - it really refers to specific `io` layout props, and only those props coming from Dato. The only "io"-ish layout left is Sentinel's, and it does _not_ require any data from Dato.

So, `layoutProps` seems very safe to remove (and as mentioned previously, somewhat necessary to make the `next`-version-parity fix easier by removing `rivet-graphql`.